src/group_theory/free_group.lean

```diff
@@ -863,4 +883,66 @@ fintype.subtype (list.to_finset $ red.enum L₁) $
 
 end reduce
 
+section metric
+
+variable [decidable_eq α]
+
+/-- The length of reduced words provides a norm on a free group. --/
+def norm (x : free_group α) : nat := x.to_word.length
+
+private lemma norm_inv_le (x : free_group α) : norm x⁻¹ ≤ norm x :=
```

Comment:
```suggestion
lemma norm_inv_le (x : free_group α) : norm x⁻¹ ≤ norm x :=
```
We generally avoid marking anything `private` unless it's extremely specific, which this isn't! You'd be surprised what comes in handy now and then.